packages/components/src/components/flyout/flyout.examples.md

```diff
@@ -78,15 +97,21 @@ p-flyout {
 import Vue from 'vue';
 import Component from 'vue-class-component'; 
 import { getFlyoutCodeSamples } from "@porsche-design-system/shared";  
+import ExampleStylesGrid from '@/pages/patterns/styles/example-grid.vue';
 import { convertMarkup } from '@porsche-design-system/storefront/src/utils/formatting.ts';
 
-@Component
+@Component({
+  components: {
+    ExampleStylesGrid
+  },
+})
 export default class Code extends Vue {
   config = { themeable: true, overflowX: 'visible' };
   flyouts = [];
   codeExample = getFlyoutCodeSamples('default');
   codeExampleSlotted = getFlyoutCodeSamples('example-slotted'); 
   codeExampleSlottedSecondary = getFlyoutCodeSamples('example-slotted-secondary');
+  codeExampleSlottedGrid = getFlyoutCodeSamples('example-slotted-grid');
```

Comment:
Those won't work because all the external grid components and/or styles are missing?

Reply:
This is just for rendering `:frameworkMarkup` in the Storefront example page.